scripts/build-docs/index.ts

```diff
@@ -447,6 +447,17 @@ const preamble =
 
 **Pro Tip**: You can define your own provider by adding \`const provider = ganache.provider({})\` to the start of any example and passing in your [startup options](https://trufflesuite.com/docs/ganache/reference/cli-options/).`);
 
+// these are the fonts that are loaded when you navigate to ganache. It's possible others could be
+// loaded if users use the page, but these are the initial ones at time of writing. To verify this
+// is still true, load ganache.dev with devtools open, and check the `Font` tab within the Network
```

Comment:
I have a couple extras on Chome on macOs:
```
https://fonts.gstatic.com/s/opensans/v35/memQYaGs126MiZpBA-UFUIcVXSCEkx2cmqvXlWq8tWZ0Pw86hd0Rk5hkWVAewIMUdjFXmQ.woff2
https://fonts.gstatic.com/s/opensans/v35/memvYaGs126MiZpBA-UvWbX2vVnXBbObj2OVTS-mu0SC55I.woff2
https://fonts.gstatic.com/s/sharetechmono/v15/J7aHnp1uDWRBEqV98dVQztYldFcLowEFA87Heg.woff2
https://fonts.gstatic.com/l/font?kit=7Au7p_IgjDKdCRWuR1azpmQIElsO02s2tozWoxjt&skey=3d5373e9fc70eefe&v=v14
https://cdnjs.cloudflare.com/ajax/libs/monaco-editor/0.28.0/min/vs/base/browser/ui/codicons/codicon/codicon.ttf
```

Reply:
Turns out google loads different fonts for each browser :-(

From https://developers.google.com/fonts/docs/technical_considerations:

> What is the Google Fonts API serving?
> When a browser sends a request for a Fonts API stylesheet (as specified in a <link> tag in your web page), the Fonts API serves a stylesheet generated for the specific user agent making the request.

So I should probably remove the preload links :-(

Reply:
I've removed these google font preloads and added one for codicon